src/main/kotlin/com/zepben/evolve/database/sqlite/tables/iec61968/common/TableStreetAddresses.kt

```diff
@@ -14,5 +14,13 @@ import com.zepben.evolve.database.sqlite.tables.Column.Nullable.NOT_NULL
 abstract class TableStreetAddresses : TableTownDetails() {
 
     val POSTAL_CODE = Column(++columnIndex, "postal_code", "TEXT", NOT_NULL)
+    val PO_BOX = Column(++columnIndex, "po_box", "TEXT", NOT_NULL)
+    val BUILDING_NAME = Column(++columnIndex, "building_name", "TEXT", Column.Nullable.NULL)
+    val FLOOR_IDENTIFICATION = Column(++columnIndex, "floor_identification", "TEXT", Column.Nullable.NULL)
+    val NAME = Column(++columnIndex, "name", "TEXT", Column.Nullable.NULL)
+    val NUMBER = Column(++columnIndex, "number", "TEXT", Column.Nullable.NULL)
+    val SUITE_NUMBER = Column(++columnIndex, "suite_number", "TEXT", Column.Nullable.NULL)
+    val TYPE = Column(++columnIndex, "type", "TEXT", Column.Nullable.NULL)
+    val DISPLAY_ADDRESS = Column(++columnIndex, "display_address", "TEXT", Column.Nullable.NULL)
```

Comment:
These columns are marked as `NULL` but the fields of the class are marked as non-null. If you change the class to allow nulls can you please import `NULL` to allow the removal of `Column.Nullable.` from each column def, otherwise update the columns to `NOT_NULL`

Reply:
TownDetail.kt/TableTownDetails.kt seems to have the same problem. Should I make those `NOT_NULL` as well?

Reply:
New revision makes `TownTableDetails` columns `NOT_NULL`.